src/libdrakvuf/os.c

```diff
@@ -374,3 +374,27 @@ status_t drakvuf_find_mmvad(drakvuf_t drakvuf, addr_t eprocess, addr_t vaddr, mm
 
     return 0;
 }
+
+status_t drakvuf_get_wow_context(drakvuf_t drakvuf, addr_t ethread, addr_t* wow_ctx)
+{
+    if ( drakvuf->osi.get_wow_context )
+        return drakvuf->osi.get_wow_context(drakvuf, ethread, wow_ctx);
+
+    return 0;
+}
+
+status_t drakvuf_get_stack32(drakvuf_t drakvuf, drakvuf_trap_info_t* info, addr_t* stack_ptr)
```

Comment:
I guess we should call these `get_userstack`. Also a comment above would be nice saying "fetch userland esp/rsp from inside kernel".